lib/screens/scanner/scanner_screen.dart

```diff
@@ -177,70 +200,47 @@ class _ScannerScreenState extends State<ScannerScreen> {
     if (_pairingState != ScannerScreenPairingState.ready) return;
 
     _updatePairingState(ScannerScreenPairingState.inProgress);
+    setState(() {
```

Comment:
We use separate function to update the values using setState, example _updatePairingState, _updateRecoveryState

Reply:
Its seem unnecessary, this state is only set once